sequencer/api/node.toml

```diff
@@ -11,3 +11,7 @@ DOC = "Get the stake table for the given epoch"
 PATH = ["validators/:epoch_number"]
 ":epoch_number" = "Integer"
 DOC = "Get the validators map for the given epoch."
+
+[route.get_block_reward]
```

Comment:
Should this take an epoch number in case we want to update it? We can also do that later. I'm not quite sure what we need this API for hence my question.